pkg/frontend/openshiftcluster_putorpatch.go

```diff
@@ -136,7 +137,7 @@ func (f *frontend) _putOrPatchOpenShiftCluster(ctx context.Context, log *logrus.
 		// Persist identity URL and tenant ID only for managed/workload identity cluster create
 		// We don't support updating cluster managed identity after cluster creation
 		// TODO - use a common function to check if the cluster is a managed/workload identity cluster
```

Comment:
Nit: This line with the `TODO` comment can be removed since this PR implements the change it describes.